posts/2023-05-24-api-automation-testing-tools-comparison.adoc

```diff
@@ -0,0 +1,320 @@
+= API Automation Testing Tools, a comparison
+TehreemFatima07
+v1.0, 2023-05-24
+:title: API Automation Testing Tools, a comparison
+:imagesdir: ../media/2023-05-24-api-automation-testing-tools-comparison
+:lang: en
+:tags: [api, test automation, testing techniques]
+
+
+Internal projects at Lunatech play a crucial role in fostering growth, innovation, and knowledge-sharing within employees. These projects serve as the sandbox where new joiners are given the opportunity to explore, learn, and contribute in a hands-on manner.
+++++
+</br>
+++++
+Lunagraph is one of the internal projects that will revolutionize how we handle three crucial elements: People, Clients, and Projects. This innovative solution serves as the "source of truth" for these key elements, providing a reliable and consolidated platform that offers immense value to our internal clients.
+++++
+</br>
+++++
+Lunagraph's vision is to be able to reliably service internal clients with a single API integration, previously many apps were used for this integration. One of the task for this project is to have automated testing of APIs for faster test maintenance and refactoring.
+
+=== What is an API Testing?
```

Comment:
Should there be maybe a section for explaining why we chose these tools in particular? As some of thema re more leaning to be UI automation frameworks/testing tools, so maybe it could be worth sentence or two explaining the thought process why these in particular.
Maybe in this section, or new section altogether?